nexus-repository-conan/src/main/java/org/sonatype/repository/conan/upgrade/ConanUpgrade_1_1.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.conan.upgrade;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Provider;
+import javax.inject.Singleton;
+
+import org.sonatype.nexus.common.upgrade.DependsOn;
+import org.sonatype.nexus.common.upgrade.Upgrades;
+import org.sonatype.nexus.orient.DatabaseInstance;
+import org.sonatype.nexus.orient.DatabaseInstanceNames;
+import org.sonatype.nexus.orient.DatabaseUpgradeSupport;
+import org.sonatype.nexus.orient.OIndexNameBuilder;
+import org.sonatype.repository.conan.internal.AssetKind;
+import org.sonatype.repository.conan.internal.ConanFormat;
+
+import com.orientechnologies.orient.core.db.document.ODatabaseDocumentTx;
+import com.orientechnologies.orient.core.db.record.OIdentifiable;
+import com.orientechnologies.orient.core.index.OIndex;
+import com.orientechnologies.orient.core.record.impl.ODocument;
+import com.orientechnologies.orient.core.sql.query.OSQLSynchQuery;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+import static org.sonatype.nexus.repository.storage.AssetEntityAdapter.P_ASSET_KIND;
+
+/**
+ * @since 1.0.0
+ */
+@Named
+@Singleton
+@Upgrades(model = ConanModel.NAME, from = "1.0", to = "1.1")
+@DependsOn(model = DatabaseInstanceNames.COMPONENT, version = "1.14", checkpoint = true)
+@DependsOn(model = DatabaseInstanceNames.CONFIG, version = "1.8", checkpoint = true)
```

Comment:
appears there are no changes made to the config database (only the component database), so i don't think you need `checkpoint = true` here

Reply:
fixed https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/95/commits/9938f7c67cac959dbbf6292c9271250ee6812827

Reply:
You know, after this changes IT is failing in CI and my local env.
I got
org.sonatype.nexus.common.stateguard.InvalidStateException: Invalid state: NEW; allowed: [STARTED]

Maybe you make mistake about checkpoint=false ?
With checkpoint=true, everything ok

Reply:
Thanks, removed checkpoint = true from CONFIG
but checkpoint = true  in COMPONENT setted

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/95/commits/b669860e4a223502adf1793e400af5d579cf2c76